fastlane/Fastfile

```diff
@@ -1119,6 +1119,46 @@ platform :ios do
     trigger_action_in_circle_ci(action: 'bump', repo_name: REPO_NAME)
   end
 
+  desc "Trigger bump SDK in RC mobile app"
+  lane :trigger_bump_sdk_in_rc_mobile_app do
+    require 'rest-client'
+
+    version = current_version_number
+
+    circle_token = ENV["RC_APP_CIRCLE_CI_TOKEN"]
+    UI.user_error!("Please set the RC_APP_CIRCLE_CI_TOKEN environment variable") unless circle_token
+
+    definition_id = ENV["RC_APP_CIRCLE_CI_PIPELINE_DEFINITION_ID"]
+    UI.user_error!("Please set the RC_APP_CIRCLE_CI_PIPELINE_DEFINITION_ID environment variable") unless definition_id
+
+    project_slug = ENV["RC_APP_CIRCLE_CI_PROJECT_SLUG"]
+    UI.user_error!("Please set the RC_APP_CIRCLE_CI_PROJECT_SLUG environment variable") unless project_slug
+
+    # Make request
+    headers = {"Circle-Token": circle_token, "Content-Type": "application/json", "Accept": "application/json"}
+    data = {
+      definition_id: definition_id,
+      config: { branch: "main" },
+      checkout: { branch: "main" },
```

Comment:
I guess we could make the branch a parameter... But I think it's ok to just have main for now and we could add it later when needed.

Reply:
Right. I actually thought about it, but "main" feels like static enough, and it's hardcoded everywhere else. But I agree, we can make it a parameter later if we need.
Thanks for the suggestion!